src/view-factory.js

```diff
@@ -158,6 +169,14 @@ function applyInstructions(containers, element, instruction, controllers, bindin
     return;
   }
 
+  if (instruction.letElement) {
```

Comment:
@EisenbergEffect Here is the difference compared to the previous implementation, it will just be a normal `TargetInstruction` with `.letElement === true`  and thus, behaves the way it's specified in the DOM.